packages/api/src/learning-materials/queries/what-is-user-name.query.ts

```diff
@@ -0,0 +1,9 @@
+import { Query } from '@nestjs-architects/typed-cqrs';
+
+import type { UserId } from '../../users/users.types';
```

Comment:
Or.... maybe, what do you think - about placing query handler and this query in learning-materials... hmm....
Because the question is specific for learning materials, and query handler will work like an adapter.